### ISSUE TITLE
Incorrect Argument Passed to `brier_score_loss`: Predictions Used Instead of Probabilities

### ISSUE THREAD

**Issue by lshpaner** [uclamii/model_tuner#163]
Brier score loss, as per the [documentation](https://scikit-learn.org/dev/modules/generated/sklearn.metrics.brier_score_loss.html) takes in `y_true, y_proba`, but the `brier_score_loss` inside the `report_model_metrics` is currently ingesting the predictions instead of the probability for the second argument, incorrectly:

This:

```python
"Value": brier_score_loss(aggregated_y_true, aggregated_y_pred),
```

Should be this:

```python
"Value": brier_score_loss(aggregated_y_true, aggregated_y_prob),
```
